documentation/content/codingstyle.rst

```diff
@@ -193,19 +193,19 @@ Test name
 ---------
 
 Tests are an important part of kw, we only accept new features with tests, and
-we prefer bug fixes that came with tests. For trying to keep the test
+we prefer bug fixes that come with tests. For trying to keep the test
 comprehensible, we adopt the following pattern for naming a test::
 
-    target_function_name_[an_option_description]_Test
+    test_target_function_name_[an_optional_description]
 
 To better illustrate this definition, see the below example::
 
-    detect_distro_Test
+    test_detect_distro
```

Comment:
I believe that `()` is mandatory if we want the auto-detection provided by shunit.

Reply:
Yes it is! Nice catch